README.md

```diff
@@ -1,86 +1,126 @@
+### `3.5`
 
-## License of this repository
+This image is for .NET Framework 3.5 and earlier version applications.  It is based on the [Windows Server Core image](https://hub.docker.com/r/microsoft/windowsservercore/).
 
-MIT
+## Related Repos and Examples
 
-## License of .NET Framework Docker images 
+See [.NET Framework Docker samples] to get started with pre-made samples.
+
+See [.NET Framework and Docker](https://docs.microsoft.com/dotnet/articles/framework/docker/) to learn more about using .NET Framework with Docker.
+
+See the following related repos for other application types:
+
+- [microsoft/aspnet](https://hub.docker.com/r/microsoft/aspnet/) for ASP.NET Web Forms and MVC applications.
+- [microsoft/dotnet](https://hub.docker.com/r/microsoft/dotnet/) for .NET Core applications.
+- [microsoft/aspnetcore](https://hub.docker.com/r/microsoft/aspnetcore/) for ASP.NET Core applications.
+- [Windows Server Core image](https://hub.docker.com/r/microsoft/windowsservercore/) for Windows Server Core applications, such as .NET Frameworks applications.
+
+You can read more about [Windows Containers](https://msdn.microsoft.com/virtualization/windowscontainers/about/about_overview) to learn how to use Docker with Windows.
+
+# License
+
+The .NET Framework images use the same license as the ]Windows Server Core base image](https://hub.docker.com/r/microsoft/windowsservercore/), as follows:
 
 MICROSOFT SOFTWARE SUPPLEMENTAL LICENSE TERMS
 
 CONTAINER OS IMAGE
 
 Microsoft Corporation (or based on where you live, one of its affiliates) (referenced as “us,” “we,” or “Microsoft”) licenses this Container OS Image supplement to you (“Supplement”). You are licensed to use this Supplement in conjunction with the underlying host operating system software (“Host Software”) solely to assist running the containers feature in the Host Software. The Host Software license terms apply to your use of the Supplement. You may not use it if you do not have a license for the Host Software. You may use this Supplement with each validly licensed copy of the Host Software.
 
-## User Feedback
+# Supported Docker versions
+
+This image is officially supported on Docker version 1.12.2.
+
+Please see [the Docker installation documentation](https://docs.docker.com/installation/) for details on how to upgrade your Docker daemon.
+
+# User Feedback
+
+## Issues and Contributing
+
+You are invited to report issues or request features by creating a [GitHub issue](https://github.com/microsoft/dotnet-framework/issues).
+
+## Documentation
 
-Please reach out to the team with any issues or concerns through a [GitHub issue](https://github.com/Microsoft/dotnet-framework-docker/issues/new).
+You can read documentation for .NET Framework, including Docker usage in the [.NET docs](https://docs.microsoft.com/dotnet/articles/framework/). The docs are also [open source on GitHub](https://github.com/dotnet/docs). Contributions are welcome!
```

Comment:
Is the docs link correct?  Is that just the docs for .Net Core or does it include full framework?